### ISSUE TITLE
Accessibility: Last correct/incorrect has no alt text

### ISSUE THREAD

**Issue by HuongNV13** [frankkoch/moodle-mod_studentquiz#127]
Hi Frank,
We found an issue related to accessibility as below:

---------------------------------------------------------------------
The graphic which displays last correct/incorrect quiz questions is not accessible to screen reader users because it has no 'alt' attribute.

Add an appropriate alt attribute to this graphic so that screen reader users are told whether their last attempt was correct or incorrect.

Thanks,
